cranelift-codegen/meta/src/shared/instructions.rs

```diff
@@ -24,19 +25,19 @@ pub fn define(
     );
 
     // Operand kind shorthands.
-    let intcc = immediates.by_name("intcc");
-    let floatcc = immediates.by_name("floatcc");
-    let trapcode = immediates.by_name("trapcode");
-    let uimm8 = immediates.by_name("uimm8");
-    let uimm32 = immediates.by_name("uimm32");
-    let imm64 = immediates.by_name("imm64");
-    let uimm128 = immediates.by_name("uimm128");
-    let offset32 = immediates.by_name("offset32");
-    let memflags = immediates.by_name("memflags");
-    let ieee32 = immediates.by_name("ieee32");
-    let ieee64 = immediates.by_name("ieee64");
-    let boolean = immediates.by_name("boolean");
-    let regunit = immediates.by_name("regunit");
+    let intcc = &imm.intcc;
+    let floatcc = &imm.floatcc;
+    let trapcode = &imm.trapcode;
+    let uimm8 = &imm.uimm8;
+    let uimm32 = &imm.uimm32;
+    let imm64 = &imm.imm64;
+    let uimm128 = &imm.uimm128;
+    let offset32 = &imm.offset32;
+    let memflags = &imm.memflags;
+    let ieee32 = &imm.ieee32;
+    let ieee64 = &imm.ieee64;
+    let boolean = &imm.boolean;
+    let regunit = &imm.regunit;
```

Comment:
These re-assignment lists are just another level of indirection when trying to look up the code; can we just do `&imm.ieee32`, e.g., where it is used?

Reply:
Yes, good idea!